### PR TITLE
Added Enhanced Media Gallery docs 

### PR DESCRIPTION
**Pull request by gabrieldagama**
## Purpose of this pull request

This pull request (PR) adds documentation for the Enhanced Media Gallery feature.

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/configuration/advanced/system.html
- https://docs.magento.com/m2/ce/user_guide/cms/content-menu.html
- https://docs.magento.com/m2/ce/user_guide/cms/enhanced-media-gallery.html
- https://docs.magento.com/m2/ce/user_guide/cms/enhanced-media-gallery-image-management.html
- https://docs.magento.com/m2/ce/user_guide/cms/enhanced-media-gallery-folder-management.html
- https://docs.magento.com/m2/ce/user_guide/cms/enhanced-media-gallery-license-image.html

## Links to Magento source code or PRs

https://github.com/magento/adobe-stock-integration

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cms/enhanced-media-gallery-license-image.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: License an Adobe Stock Image
+---
+
+[Enhanced Media Gallery]({% link cms/enhanced-media-gallery.md %}) provides a direct integration with Adobe Stock, allowing the user to license their images directly from the gallery page.
```
Comment (sivaschenko) [inline src/cms/enhanced-media-gallery-license-image.md:5]: Please mention that this functionality requires Adobe Stock Integration to be enabled and configured

File: src/cms/enhanced-media-gallery.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Merchants can use the Enhanced Media Gallery to organize and manage their media files on the server. The Enhanced Media Gallery contains the same functionalities as the Media Storage with an improved user interface and a closer integration with [Adobe Stock][adobe-stock].
+
+![Enhanced Media Gallery Grid]({% link images/images/enhanced-media-gallery/gallery-grid.png %}){: .zoom}
+
+## Prerequisites
+
+The Enhanced Media Gallery requires Magento 2.4 or newer. 
+
+## Enable the Enhanced Media Gallery
```
Comment (sivaschenko) [inline src/cms/enhanced-media-gallery.md:13]: Please add the information covering the:
 - Assets and asset-content relations synchronization and corresponding CLI commands
 - View details slide panel functionality (covering the information and actions, ensure to feature the "Used In" information that is new to the Magento admin user)

Comment (gabrieldagama) [reply src/cms/enhanced-media-gallery.md:13]: As discussed the CLI commands will be addressed on the DevDocs and we will link this later.

**Comment by leslietilling** [comment]
At the request of the PM, we will not use the term "Enhance Media Gallery". I will create a new PR to update the terminology.

**Comment by tkacheva** [comment]
@sivaschenko please use "new media gallery"

**Comment by leslietilling** [comment]
> @sivaschenko please use "new media gallery"

This PR made updates for that terminology: https://github.com/magento/merchdocs/pull/591

**Pull request merged**